backend/storage/addons/discordwebhook/conf.yml

```
@@ -0,0 +1,17 @@
+plugin:
+  name: "Discord Webhook"
+  identifier: "discordwebhook"
+  description: "Discord Webhook plugin for mythicalclient to send messages to a Discord Webhook!"
+  version: "1.0.0"
+  author: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)